Dockerfile

```diff
@@ -6,17 +6,19 @@ COPY . .
 RUN make install
 
 ############# base image
-FROM alpine:3.13.7 AS base
+FROM gcr.io/distroless/static-debian11:nonroot AS base
```

Comment:
In https://github.com/gardener/gardener/pull/6159 we changed the port of the gardener-apiserver from 443 to 8443
> so that it can be run as nonroot and avoid setting additional capabilities to that binary.

I see that admission-gcp is using 443 as well. Won't this break the admission-gcp image?

Reply:
Isn't [this](https://github.com/gardener/gardener-extension-provider-gcp/blob/e232602ca54048512a2c94c68ce5237a449d6741/charts/gardener-extension-admission-gcp/values.yaml#L26) the port the webhook is started on? The service port [here](https://github.com/gardener/gardener-extension-provider-gcp/blob/master/charts/gardener-extension-admission-gcp/charts/runtime/templates/service.yaml#L13) is OK to be left 443.

Reply:
Okay, thanks for explaining!